nextmove/src/main/java/no/difi/meldingsutveksling/nextmove/DpvConversationResource.java

```diff
@@ -1,43 +1,60 @@
 package no.difi.meldingsutveksling.nextmove;
 
-import com.google.common.collect.Maps;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 import no.difi.meldingsutveksling.ServiceIdentifier;
 
 import javax.persistence.DiscriminatorValue;
 import javax.persistence.Entity;
 import java.time.LocalDateTime;
+import java.util.Map;
 
 @Entity
 @DiscriminatorValue("DPV")
+@NoArgsConstructor
 @Data
 public class DpvConversationResource extends ConversationResource {
 
     private String messageTitle;
     private String messageContent;
 
-    DpvConversationResource() {}
-
     private DpvConversationResource(String conversationId, String senderId, String receiverId) {
-        super(conversationId, senderId, receiverId, ServiceIdentifier.DPV, LocalDateTime.now(), Maps.newHashMap(), Maps.newHashMap());
+        super(conversationId, senderId, receiverId, ServiceIdentifier.DPV);
+    }
+
+    private DpvConversationResource(String conversationId,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS00107)